DNN Platform/Website/DotNetNuke.Website.csproj

```diff
@@ -1275,6 +1275,7 @@
     <Content Include="Providers\DataProviders\SqlDataProvider\09.12.01.SqlDataProvider" />
     <Content Include="Install\Config\09.13.04.config" />
     <Content Include="Providers\DataProviders\SqlDataProvider\09.13.04.SqlDataProvider" />
+    <Content Include="Providers\DataProviders\SqlDataProvider\09.13.08.SqlDataProvider" />

```

Comment:
I don't see this file in the PR, should this be here?

Reply:
It was pre-existing but never got added to the project file https://github.com/dnnsoftware/Dnn.Platform/blob/develop/DNN%20Platform/Website/Providers/DataProviders/SqlDataProvider/09.13.08.SqlDataProvider